learning-journal-basic/learning_journal_basic/routes.py

```diff
@@ -0,0 +1,9 @@
+"""Hold route info broken out of __init__."""
+
+def includeme(config):
+    """This function adds routes to pyramid's configurator."""
+    config.add_static_view('static', 'static', cache_max_age=3600)
+    config.add_route('home', '/')
+    config.add_route('new', '/journal/new-entry')
+    config.add_route('detail', '/journal/1')
+    config.add_route('edit', '/journal/1/edit-entry')
```

Comment:
This and the detail route should be using a route pattern, not just a bare number